### PR TITLE
Jakarta XML Binding 2.3

### PR DESCRIPTION
**Pull request by lukasj**
Signed-off-by: Lukas Jungmann <lukas.jungmann@oracle.com>

PR#1:
- [x] A directory in the form wombat/x.y where x.y is the release major.minor version, and the directory contains the following.
- [x] Specification PDF in the form of wombat_x.y.pdf
- [x] Specification HTML in the form of wombat_x.y.html
- [x] A specification page named _index.md following the template at:
      https://github.com/jakartaee/specification-committee/blob/master/spec_page_template.md
- [x] For a Release Review, a summary that a Compatible Implementation
      is complete, passes the TCK, and that the TCK includes sufficient
      coverage of the specification. The TCK users guide MUST include
      the instructions to run the compatible implementations used to
      validate the release.  Instructions MAY be by reference.
- [x] The URL of the OSSRH staging repository for the api, javadoc:
      https://oss.sonatype.org/content/repositories/staging/jakarta/xml/bind/jakarta.xml.bind-api/2.3.3/
- [x] The URL of the staging directory on downloads.eclipse.org for the proposed EFTL TCK binary:
      http://download.eclipse.org/ee4j/jakartaee-tck/jakartaee8-eftl/promoted/eclipse-jaxb-tck-2.3.0.zip
- [x] The URL of the compatibility certification request issue:
      https://github.com/eclipse-ee4j/jaxb-api/issues/115

Include the following in PR#2:
- [x] Specification JavaDoc in the wombat/x.y/apidocs directory.

### CODE REVIEW

**Comment by lukasj** [comment]
this should be now ready for ballot...

**Comment by lukasj** [comment]
@dblevins so this (as few others) PR was approved and staged artifacts will expire soon (some  already did (again), others will in ~week). As I really don't want to waste more time then is necessary on this - what do I have to do, to move this and other related PRs forward?

**Comment by kwsutter** [comment]
Just putting the "hold" tag on this one to be consistent with the others in this same class -- Java SE 11 drops.  We're making some good progress on the Platform project to understand where we go with these in Jakarta EE.  Thanks for your patience.

File: xml-binding/2.3/_index.md [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+date: 2019-07-22T17:54:14+0200
+summary: "First release for Jakarta EE 8"
+---
+The Jakarta XML Binding provides an API and tools that automate the mapping
+between XML documents and Java objects.
+
+* [Jakarta XML Binding 2.3 Specification Document](./jakarta.xml.bind-spec-2.3.pdf) (PDF)
+* [Jakarta XML Binding 2.3 Specification Document](./jakarta.xml.bind-spec-2.3.html) (HTML)
+* [Jakarta XML Binding 2.3 Javadoc](./apidocs)
+* [Jakarta XML Binding 2.3 TCK](http://download.eclipse.org/ee4j/jakartaee-tck/8.0.1/promoted/jaxb-tck-2.3_latest.zip)
```
Comment (kwsutter) [inline xml-binding/2.3/_index.md]: Did something change with our TCK publishing process?  This URL doesn't work.  Actually, all of the sub-directories under the top level directory have changed...  https://download.eclipse.org/ee4j/jakartaee-tck/.  This comment probably applies to all of the Java SE PRs.

Comment (bshannon) [reply xml-binding/2.3/_index.md]: This is an (incorrect) reference to the staging directory.
The final TCK will be published under downloads.eclipse.org/jakarta/xml-binding/2.3/...

**Comment by lukasj** [comment]
@kwsutter @bshannon I would need refreshed JAF 1.2.2 binaries in order to refresh jaxb-api (and in turn also saaj-api and jax-ws-api). This would bring us having all these "SE APIs" in sync. Alternatively, I can make affected APIs dependent on JAF 1.2.1. What is your preference?

**Comment by bshannon** [comment]
> @kwsutter @bshannon I would need refreshed JAF 1.2.2 binaries in order to refresh jaxb-api (and in turn also saaj-api and jax-ws-api). This would bring us having all these "SE APIs" in sync. Alternatively, I can make affected APIs dependent on JAF 1.2.1. What is your preference?

Refreshed JAF 1.2. binaries are available in the [jakarta Nexus staging repo](https://jakarta.oss.sonatype.org/content/repositories/staging/jakarta/activation/jakarta.activation-api/1.2.2/).

**Comment by lukasj** [comment]
TODOs: upload refreshed binaries, fix TCK URL

**Comment by lukasj** [comment]
binaries were refreshed

File: xml-binding/2.3/_index.md [hunk 2]
```diff
@@ -0,0 +1,37 @@
+---
+title: "Jakarta XML Binding Specification 2.3"
+date: 2019-07-22T17:54:14+0200
+summary: "Initial release of Jakarta XML Binding"
+---
+The Jakarta XML Binding provides an API and tools that automate the mapping
+between XML documents and Java objects.
+
+* [Jakarta XML Binding 2.3 Specification Document](./jakarta.xml.bind-spec-2.3.pdf) (PDF)
```
Comment (kwsutter) [inline xml-binding/2.3/_index.md]: Per the comments on the other related PRs, the name of the Spec Doc should be xml-binding-spec-2.3.pdf per our [naming doc](https://github.com/jakartaee/specification-committee/blob/master/names.adoc).

File: xml-binding/2.3/_index.md (hunk 1 above)
Comment (lukasj) [reply xml-binding/2.3/_index.md]: done

**Comment by kwsutter** [comment]
- [ ] The designated specification committee representative adds this final checklist to the main PR.
 - [ ] The specification committee representative merges the specification and apidocs PRs.
 - [ ] The designated specification committee representative calculates the staged EFTL TCK signature and promotes it to the committee download area
  using the https://ci.eclipse.org/jakartaee-spec-committee/job/promote-release/ job.
 - [ ] The specification project member who created the staging release promotes the specification api jars to maven central. An example release job script can be found here https://wiki.eclipse.org/MavenReleaseScript.
 - [ ] The EMO updates the specification page with the ballot results.
This list goes on the committed spec index page.
 - [ ] The specification project team should go through the merged spec website page to verify all the links are valid.
 - [ ] The specification project team should approve the compatibility request.
 - [ ] The compatible implementation project/vendor MUST send an email tck@eclipse.org for approval of the compatible implementation for trademark usage.
 - [ ] The specification project team should merge any final release branch as appropriate for the branch management for the project.

**Comment by lukasj** [comment]
@kwsutter unchecked items were done

**Pull request merged**